### DISCUSSION TITLE
v2.2.0

### DISCUSSION THREAD

**Discussion by Torxed** [archlinux/archinstall#528] [Show and tell]
**Known Issues:**
 * Partitioning still has issues with certain layouts.
   **Workaround:** Manually creating partitions with supported filesystems, mounting them at `/mnt` and selecting *"use /mnt as is"*, or skip the mount step and select *"Re-use partitions"*  will solve most issues. Wiping the desired drive before running archinstall is also a workaround for now but remember to backup your data! Work is already underway to correct this, but will not make it in time for this release, follow the progress here: #426
 * Dual-Boot with Windows works, but there's an issue with Windows creating a very small `/boot` partition which causes issues, please track the progress here: #527
 * pypi.org had a version collision from a [previously uploaded](https://pypi.org/project/archinstall/2.2.0/) *(wrongly named)* version. Therefor there's a **v2.2.0[.post1](https://pypi.org/project/archinstall/2.2.0.post1/)** release exclusively for pypi.org

Top New Features
==============

 * GRUB & MBR support
 * `ucode` for Intel and AMD is automatically installed
 * Moved from `FreeNode` to `libera.chat` *(on IRC)*
 * Selecting Kernels
 * Loading configuration from a pre-defined config (re-run your installs easily!)

<hr /><em>This discussion was created from the release <a href='https://github.com/archlinux/archinstall/releases/tag/v2.2.0'>v2.2.0</a>.</em>

**Comment by Torxed** [comment]
Just a heads up to the general population, we do have an issue uploading this to pypi because of a version collision previously uploaded. This affects `v2.2.0` only. I'm not sure we can sort it out, but we might have to deviate and release a patch version on pypi.org exclusively to get around the version blockage.

Read more here: https://stackoverflow.com/a/13919964/929999
And here: https://github.com/pypa/packaging-problems/issues/74

**Reply by Lunarequest** [reply]
you can delete the old one and push the new version

**Reply by Torxed** [reply]
Sadly after deleting, there's no way of pushing the same file name (version) to pypi.org since ~2014, you'll end up getting:
> HTTPError: 400 Client Error: This filename has previously been used, you should use a different version.

**Reply by Torxed** [reply]
On pypi.org exclusively, there's now a **v2.2.0[.post1](https://pypi.org/project/archinstall/2.2.0.post1/)** release.
To get around the `400 Client Error`.

**Reply by m1ten** [reply]
does v2.2.0-2 work?

**Reply by Torxed** [reply]
It should be yea, why? :)

**Reply by m1ten** [reply]
I was just wondering if `-number` could have been used instead of `.post1`.

**Reply by Torxed** [reply]
Aah ^^ possibly, since that's a Arch specific thing I wasn't sure if it would cause issues there some how :) but good idea
